### PR TITLE
Updating README file for dgraph-js v2.0 changes

### PR DESCRIPTION
**Pull request by prashant-shahi**
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph-js/65)

### CODE REVIEW

**Review by danielmai — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph-js/65)* status: all files reviewed, 4 unresolved discussions (waiting on @paulftw and @prashant-shahi)

___
*[README.md, line 71 at r1](https://reviewable.io/reviews/dgraph-io/dgraph-js/65#-LnsQc9k4lpkh7Opk-7Z:-LnsQc9k4lpkh7Opk-7_:bq1okd3) ([raw file](https://github.com/dgraph-io/dgraph-js/blob/4dc0f8938855d5b2ce63836a331ce24cfa544903/README.md#L71)):*
> ```
> One of the most important API breakages
> ```
Is this the only API breakage, or are there others? If this is the only one, then say there's only one instead of "one of".
___
*[README.md, line 72 at r1](https://reviewable.io/reviews/dgraph-io/dgraph-js/65#-LnsQC722ObC_chHpg4N:-LnsQC722ObC_chHpg4O:byqz067) ([raw file](https://github.com/dgraph-io/dgraph-js/blob/4dc0f8938855d5b2ce63836a331ce24cfa544903/README.md#L72)):*
> ```
> an `messages.Assigned`
> ```
a `messages.Assigned`
___
*[README.md, line 73 at r1](https://reviewable.io/reviews/dgraph-io/dgraph-js/65#-LnsQFMXAaOkZuRlJek6:-LnsQFMXAaOkZuRlJek7:by1y673) ([raw file](https://github.com/dgraph-io/dgraph-js/blob/4dc0f8938855d5b2ce63836a331ce24cfa544903/README.md#L73)):*
> ```
> an `messages.Response`
> ```
a `message

File: README.md [hunk 1]
```diff
@@ -170,7 +183,20 @@ not run conflict detection over the index, which would decrease the number of
 transaction conflicts and aborts. However, this would come at the cost of potentially
 inconsistent upsert operations.
 
-### Run a query
+Mutation can be run using `txn.doRequest` as well.
+
+```js
+const mu = new dgraph.Mutation();
+mu.setSetJson(q);
```
[the remark this answers is not in the document]
Comment (prashant-shahi) [reply README.md:190]: It is not a typo. Like you can see in the example [here](https://github.com/dgraph-io/dgraph-js/blob/71699030eadd590bec77ba0b0a178028395c8320/examples/simple/index.js#L70).

**Review by prashant-shahi — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph-js/65)* status: all files reviewed, 4 unresolved discussions (waiting on @danielmai, @paulftw, @prashant-shahi, and @pullrequest[bot])

___
*[README.md, line 71 at r1](https://reviewable.io/reviews/dgraph-io/dgraph-js/65#-LnsQc9k4lpkh7Opk-7Z:-Lns_LDaC6cLIw9uJhH_:biiz7cn) ([raw file](https://github.com/dgraph-io/dgraph-js/blob/4dc0f8938855d5b2ce63836a331ce24cfa544903/README.md#L71)):*
<details><summary><i>Previously, danielmai (Daniel Mai) wrote…</i></summary><blockquote>

> ```
> One of the most important API breakages
> ```
Is this the only API breakage, or are there others? If this is the only one, then say there's only one instead of "one of".
</blockquote></details>

This seems to be the only external API change. Confirmed with Aman. 
I will be making the changes asked.

**Review by prashant-shahi — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph-js/65)* status: 0 of 1 files reviewed, 4 unresolved discussions (waiting on @danielmai, @paulftw, and @pullrequest[bot])

___
*[README.md, line 71 at r1](https://reviewable.io/reviews/dgraph-io/dgraph-js/65#-LnsQc9k4lpkh7Opk-7Z:-Lnsa4uo1Be3vXLAzuHj:b-896fix) ([raw file](https://github.com/dgraph-io/dgraph-js/blob/4dc0f8938855d5b2ce63836a331ce24cfa544903/README.md#L71)):*
<details><summary><i>Previously, prashant-shahi (Prashant Shahi) wrote…</i></summary><blockquote>

This seems to be the only external API change. Confirmed with Aman. 
I will be making the changes asked.
</blockquote></details>

Done.
___
*[README.md, line 72 at r1](https://reviewable.io/reviews/dgraph-io/dgraph-js/65#-LnsQC722ObC_chHpg4N:-Lnsa4aJ-5tcq7xk-1n5:b-896fix) ([raw file](https://github.com/dgraph-io/dgraph-js/blob/4dc0f8938855d5b2ce63836a331ce24cfa544903/README.md#L72)):*
<details><summary><i>Previously, danielmai (Daniel Mai) wrote…</i></summary><blockquote>

> ```
> an `messages.Assigned`
> ```
a `messages.Assigned`

</blockquote></details>

Done.
___
*[README.md, line 73 at r1](https://reviewable.io/reviews/dgraph-io/dgraph-js/65#-LnsQFMXAaOkZuRlJek6:-Lnsa4D29TystdpC2S3a:b-896fix) ([raw file](https://github.com/dgraph-io/dgraph-js/blob/4dc0f8938855d5b2ce63836a331ce24cfa544903/README.md#L73)):*
<details><summary><i>Previously, danielmai (Daniel Mai) wrote…</i></summary><blockquote>

> ```
> an `messages.Response`
> ```
a `message
</blockquote></details>

Done.

**Pull request merged**